lib/subcommands/ui.js

```diff
@@ -46,6 +46,8 @@ commands.setAppearance = async function setAppearance (appearance) {
  *
  * @this {import('../simctl').Simctl}
  * @return {Promise<string>} the contrast configuration value.
+ *                           Possible return value is 'enabled', 'disabled',
```

Comment:
Prefer to use `@since` to provide the minimum Xcode version

Reply:
I haven't tested such as "minimum" but checked this was available in xcode 15 as well. Let me leave note as well and use `since` with xcode 15